modules/core/src/test/scala/io/strongtyped/active/slick/EntityActionsBeforeInsertUpdateTest.scala

```diff
@@ -15,6 +13,8 @@ class EntityActionsBeforeInsertUpdateTest
 
   it should "return an error if beforeInsert is not successful" in {
 
+    import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
If import it in global scope. Method `beforeInsert` and `beforeUpdate` in line 88 and line 97 will compile fail with
```scala
active-slick/modules/core/src/test/scala/io/strongtyped/active/slick/EntityActionsBeforeInsertUpdateTest.scala:97: ambiguous implicit values:
[error]  both value exc of type scala.concurrent.ExecutionContext
[error]  and value global in object Implicits of type => scala.concurrent.ExecutionContext
[error]  match expected type scala.concurrent.ExecutionContext
[error]       findById(id).flatMap { oldModel =>
[error]                            ^
[error] one error found
```
in scala 2.12, only scala 2.11 is fine.